### ISSUE TITLE
Fix support for additional iOS build configurations

### ISSUE THREAD

**Issue by dalewking** [icerockdev/moko-resources#769]
So we added an additional configuration to the iOS side, but it was not working with task to copy framework resources to the app and the app crashed because the resources were not in the app.

To support iOS configurations other than DEBUG and RELEASE the cocoapods plugin provides the xcodeConfigurationToNativeBuildType property so you can map configurations to DEBUG and RELEASE.

Moko resources copy resources task is honoring this mapping so if the mapping is setup it will honor it.

The issue however is that the copy task is configured as soon as the moko resources plugin is applied which is before the xcodeConfigurationToNativeBuildType is setup.

The workaround that I had to use is to not apply the plugin in the plugins block:
```
plugins {
    kotlin("native.cocoapods")
    alias(libs.plugins.moko.resources) apply false
   ...
}
```

and then apply it later in the file after we configure the mapping.

The request is to delay the setup of the task so that the xcodeConfigurationToNativeBuildType mapping can be confgured without the workaround

**Comment by Alex009** [comment]
hi. thanks for report. it's same issue as here - https://github.com/icerockdev/moko-resources/issues/762
also in #762 i describe another workaround too.

we will fix it

**Comment by Alex009** [comment]
will be available in 0.24.3 release
